packages/angular/package.json

```diff
@@ -37,13 +37,15 @@
     "typescript": "3.7.5"
   },
   "scripts": {
-    "build": "run-p build:es5 build:esm",
+    "build": "run-p build:cjs build:esm",
+    "build:cjs": "tsc -p tsconfig.cjs.json",
     "build:dev": "run-s build",
-    "build:es5": "tsc -p tsconfig.build.json",
+    "build:es5": "yarn build:cjs # *** backwards compatibility - remove in v7 ***",
     "build:esm": "tsc -p tsconfig.esm.json",
-    "build:watch": "run-p build:es5:watch build:esm:watch",
+    "build:watch": "run-p build:cjs:watch build:esm:watch",
+    "build:cjs:watch": "tsc -p tsconfig.cjs.json -w --preserveWatchOutput",
     "build:dev:watch": "run-s build:watch",
-    "build:es5:watch": "tsc -p tsconfig.build.json -w --preserveWatchOutput",
+    "build:es5:watch": "yarn build:cjs:watch # *** backwards compatibility - remove in v7 ***",
```

Comment:
I don't think there is any place that uses them directly, let's just remove.

Reply:
Could do, but given that there's no harm in leaving it, any reason not to err on the side of caution?

Reply:
I guess if we are going back to standardize all the `package.json` eventually, we can get away with leaving it here for now.